packaging/format-docs.py

```diff
@@ -81,14 +93,21 @@ def format_docs(version, collectionName, types):
                         elif 'Require' in attributes:
                             defaultValue = '*(required)*'
 
-                        print(f'| {prop["PropertyName"]} | {defaultValue} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {defaultValue} | {typeName} | {prop["Description"]} |')
                     else:
-                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {typeName} | {prop["Description"]} |')
+
+    if len(relatedEnums) > 0:
+        print('\n# Related value types:\n')
```

Comment:
```suggestion
        print('\n## Related value types:\n')
```

To match namespace formatting. Nothing uses just one hash so it looks very out of place

Reply:
I was rather thinking we could add `# Weapons`/`# Traits`/`# SpriteSequences` above those to match this one. 🤔

Reply:
Perhaps, but it would still look out of place